codesamples/factories.py

```diff
@@ -42,7 +42,7 @@ def initial_data():
             <p>Experienced programmers in any other language can pick up Python very
             quickly, and beginners find the clean syntax and indentation structure
             easy to learn.
-            <a href=\"http://docs.python.org/3.3/tutorial/\">Whet your appetite</a>
+            <a href=\"https://docs.python.org/3/tutorial/\">Whet your appetite</a>
```

Comment:
This file creates data for development purposes. No need to change.

Reply:
Why not having up-to-date links in dev too?

Reply:
The test data in codesamples/factories.py is basically for frontend developers to see the slider in action, but sure let's update it.